common/adder/adder.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component version="1.4" xmlns="http://schemas.circuit-diagram.org/circuitDiagramDocument/2012/component/xml">
+  <declaration>
+    <meta name="name" value="Adder" />
+    <meta name="minsize" value="50" />
+    <meta name="author" value="Corsaka" />
+    <meta name="guid" value="8f764035-18c6-4421-ac7f-5cbd327d4e78" />
+
+    <property name="Type" type="enum" default="Full" serialize="t" display="Type">
```

Comment:
This property doesn't seem to make a difference to how the component is displayed. It would be best to remove it since it's unused.

Reply:
Ah, shoot, I forgot to implement the half adder. Good job, me.